### PR TITLE
Refactor to unify the usages of different SkyWalking versions

### PR DESCRIPTION
**Pull request by kezhenxu94**
I'll add the test case in this PR altogether after you reviewing the basic idea of the refactoring work

### CODE REVIEW

**Review by innerpeacez — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -6,66 +6,91 @@ Apache SkyWalking Kubernetes
...
+#### Prerequisites
 
-$ helm repo add elastic https://helm.elastic.co
+```shell script
+git clone https://github.com/apache/skywalking-kubernetes
+cd skywalking-kubernetes/chart
+helm repo add elastic https://helm.elastic.co
+helm dep up skywalking
+export SKYWALKING_RELEASE_NAME=skywalking  # change the release name according to your scenario
+export SKYWALKING_RELEASE_NAMESPACE=istio-system  # change the namespace according to your scenario
```
Comment (wu-sheng) [inline README.md:27]: Why do we recommend using `istio*` namespace?

File: README.md [hunk 2]
```diff
@@ -6,66 +6,91 @@ Apache SkyWalking Kubernetes
...
+```
 
-$ helm dep up skywalking
+#### Deploy the latest SkyWalking & Elasticsearch 6 (default)
```
Comment (wu-sheng) [inline README.md:30]: I prefer to remove the default, and ask the user to set explicit versions.

File: README.md [hunk 3]
```diff
@@ -6,66 +6,91 @@ Apache SkyWalking Kubernetes
...
+  --set oap.image.tag=8.1.0-es7 \
+  --set ui.image.tag=8.1.0 \
+  --set elasticsearch.imageTag=7.5.1
+``` 
 
-$ helm repo add elastic https://helm.elastic.co
+**NOTE**: Please make sure the specified OAP image tag supports the specified Elasticsearch version. 
 
-$ helm dep up skywalking
+#### Deploy the latest SkyWalking & an existing Elasticsearch
```
Comment (wu-sheng) [inline README.md:65]:
```suggestion
#### Deploy the latest SkyWalking with an existing Elasticsearch
```

**Review by wu-sheng — COMMENTED** [review]
1. Should we consider rewriting `Structure of repository` part?
2. Should we consider adding guidance about system env value injection/setting?

File: .github/workflows/e2e.istio.yaml [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+
+env:
+  SKIP_TEST: true
+  ISTIO_VERSION: 1.7.1
+  TAG: ${{ github.sha }}
+  SCRIPTS_DIR: main/test/e2e-mesh/e2e-istio/scripts
+  SW_OAP_BASE_IMAGE: openjdk:8-jre-alpine
+
+jobs:
+  als:
```
Comment (wu-sheng) [inline .github/workflows/e2e.istio.yaml:33]: I think we need `als` tests for the main repo tag or tags? How about set up a `commit-id` matrix?

**Review by wu-sheng — COMMENTED** [review]
Do you have any suggestion about how to test agent/tracing based helm?

File: README.md [hunk 5]
```diff
@@ -6,66 +6,70 @@ Apache SkyWalking Kubernetes
...
 
-$ helm repo add elastic https://helm.elastic.co
+There are required values that you must set explicitly when deploying SkyWalking.
 
-$ helm dep up skywalking
+| name | description | example |
+| ---- | ----------- | ------- |
+| `oap.image.tag` | the OAP docker image tag | `8.1.0-es6`, `8.1.0-es7`, etc. |
+| `oap.storageType` | the storage type of the OAP | `elasticsearch`, `elasticsearch7`, etc. |
+| `ui.image.tag` | the UI docker image tag | `8.1.0`, `8.1.0`, ect. |
```
Comment (wu-sheng) [inline README.md:26]:
```suggestion
| `ui.image.tag` | the UI docker image tag | `8.1.0`, `8.1.0-es7`, ect. |
```

**Comment by wu-sheng** [comment]
> 1. Should we consider rewriting Structure of repository part?
> 2. Should we consider adding guidance about system env value injection/setting?

You missed these two.

Others are generally good.

File: README.md (hunk 5 above)
Comment (kezhenxu94) [reply README.md:26]: ui tag is not related to ES version

**Comment by kezhenxu94** [comment]
> > 1. Should we consider rewriting Structure of repository part?

This part doesn't make much sense for the users NOW and I simply remove it.

> > 2. Should we consider adding guidance about system env value injection/setting?
> 

Added

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**